interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a hash in Ruby?
 
-  Your answer:
+  Your answer: A hash is used to store a pair of key-values that are made up of keys and their values.
 
-  Researched answer:
+  Researched answer: A hash is a collection of key-value pairs that consist of unique keys and their values.
```

Comment:
While your answers in this section are correct, one-line answers would be considered the bare minimum and will not impress interviewers.  I would really recommend expanding your answers a little further to show your findings and observations.